<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnswerPage</class>
 <widget class="QWidget" name="AnswerPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="answerLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 200, 0);</string>
   </property>
   <property name="text">
    <string>Correct!</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="descriptionBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>741</width>
     <height>391</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="nextLevelButton">
   <property name="geometry">
    <rect>
     <x>659</x>
     <y>530</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Next Level</string>
   </property>
  </widget>
  <widget class="QPushButton" name="retryButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>530</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Retry</string>
   </property>
  </widget>
  <widget class="QPushButton" name="levelsButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>530</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Back to Level Select</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
